common/step_download_test.go

```diff
@@ -272,9 +263,8 @@ func TestStepDownload_download(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Bad: non expected error %s", err.Error())
 	}
-	if filepath.Ext(path) != "."+step.Extension {
-		t.Fatalf("bad: path should contain extension %s but it was %s", step.Extension, filepath.Ext(path))
-	}
+	// because of the inplace option; the result file will not be renamed
+	// sha.ova.
```

Comment:
Is this going to be an issue ? Does the file needs to have an '.ova' extension ??

Reply:
Hmm looks like a few builders set an extension on download; not sure if this test is for those situations but I found this in the docs. 

```
// Extension is the extension to force for the file that is downloaded.
// Some systems require a certain extension. If this isn't set, the
// extension on the URL is used. Otherwise, this will be forced
// on the downloaded file for every URL.
```

Reply:
The file will have the name of the inplace file, right? I think the provided file has to have the right format or packer will fail, and I believe this should happen already. (not sure)

Reply:
I think it's fair to assume users are correctly naming their files. If it's not a .ova file it isn't valid, and we shouldn't be renaming files that we use in place.